# CancerMicroBiome_Reanalyzed

These are the supplementary files and tables for our bioRxiv preprint, https://www.biorxiv.org/content/10.1101/2023.07.28.550993v1.
We are posting the files here because some of them are too large for the journal to take as supplements. The paper is under review at a major scientific journal.

For those interested in the code, all of our methods are based on free, open-source software. We ran our analysis with default KrakenUniq [1] parameters against a curated Kraken database [2] extended from EuPathDB [3], so it should be easy for others to replicate our results by following our Methods as described in the preprint.

References:
1. Breitwieser FP, Baker DN, Salzberg SL. 2018. KrakenUniq: confident and fast metagenomics classification using unique k-mer counts. Genome Biol 19:198.
2. Lu J, Salzberg SL. 2018. Removing contaminants from databases of draft genomes. PLoS Comput Biol 14:e1006277.
3. Warrenfeltz S, Basenko EY, Crouch K, Harb OS, Kissinger JC, Roos DS, Shanmugasundram A, Silva-Franco F. 2018. EuPathDB: The Eukaryotic Pathogen Genomics Database Resource. Methods Mol Biol 1757:69-113.
