# CancerMicroBiome_Reanalyzed

These are the supplementary files and tables for our [bioRxiv preprint](https://www.biorxiv.org/content/10.1101/2023.07.28.550993v1) [1], which has now been published on [mBio](https://journals.asm.org/doi/10.1128/mbio.01607-23) [2].
We are posting the files here because some of them are too large for the journal to take as supplements.

For those interested in the code, all of our methods are based on free, open-source software. We ran our analysis with default KrakenUniq [3] parameters against a curated Kraken database [4] extended from EuPathDB [5], so it should be easy for others to replicate our results by following our Methods as described in the preprint.

References:
1. Gihawi, A., Ge, Y., Lu, J., Puiu, D., Xu, A., Cooper, C.S., Brewer, D., Mihaela Pertea and Salzberg, S.L. (2023). Major data analysis errors invalidate cancer microbiome findings. bioRxiv (Cold Spring Harbor Laboratory). doi:https://doi.org/10.1101/2023.07.28.550993.
2. Gihawi, A., Ge, Y., Lu, J., Puiu, D., Xu, A., Cooper, C.S., Brewer, D.S., Pertea, M. and Salzberg, S.L., 2023. Major data analysis errors invalidate cancer microbiome findings. Mbio, pp.e01607-23.
3. Breitwieser FP, Baker DN, Salzberg SL. 2018. KrakenUniq: confident and fast metagenomics classification using unique k-mer counts. Genome Biol 19:198.
4. Lu J, Salzberg SL. 2018. Removing contaminants from databases of draft genomes. PLoS Comput Biol 14:e1006277.
5. Warrenfeltz S, Basenko EY, Crouch K, Harb OS, Kissinger JC, Roos DS, Shanmugasundram A, Silva-Franco F. 2018. EuPathDB: The Eukaryotic Pathogen Genomics Database Resource. Methods Mol Biol 1757:69-113.

Updates:
1. 16 August 2023: we have updated some of the values in Table S8 and removed more false positives; the changes are very minor.
